src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1263,6 +1263,13 @@ public void ForceRun()
         #endregion
 
         #region save/load
+        public void SaveAs(string path)
+        {
+            var ws = this.CurrentWorkspace;
+            var json = Autodesk.Workspaces.Utilities.SaveWorkspaceToJson(ws, this.LibraryServices, this.EngineController,
```

Comment:
this looks out of sync with what you have written above in the workspace file. Is that true?